xml/System.IO.Compression/DeflateStream.xml

```diff
@@ -415,10 +427,10 @@ You use this constructor when you want to specify whether compression efficiency
         <ReturnType>System.IAsyncResult</ReturnType>
       </ReturnValue>
       <Parameters>
-        <Parameter Name="buffer" Type="System.Byte[]" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;xamarinandroid-7.1;xamarinios-10.8;xamarinmac-3.0;netcore-3.0;netstandard-2.1;netcore-3.1;net-5.0;net-6.0;net-7.0;net-8.0;net-9.0" />
```

Comment:
These monikers should have been deleted from the repo in https://github.com/dotnet/dotnet-api-docs/pull/9609.